pages/dkp/kommander/2.2/projects/applications/catalog-applications/index.md

```diff
@@ -19,7 +19,11 @@ Follow these steps to upgrade an application from the DKP UI:
 
 1.  From the top menu bar, select your target workspace.
 
-1.  Select **Applications** from the sidebar menu.
+1.  From the side menu bar, select **Projects**.
```

Comment:
should we put the content around project catalog app upgrade to `/projects/applications/catalog-applications/upgrading-applications/index.md` just like the workspace catalog app one's?

Reply:
yeah I can move it! one sec

Reply:
done in 60f8e8d28d40c86f8aa7ed89503b1513013873ce